### ISSUE TITLE
Cannot open App right now

### ISSUE THREAD

**Issue by piyamuna** [iNDS-Team/iNDS#171]
**Describe the bug**
When I open the app, it close immediately

**Comment by Notjailbrokenguy** [comment]
where did you got it from?

**Comment by piyamuna** [comment]
Build ISO website sir

https://builds.io/apps/inds/

With registered and paid

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Notjailbrokenguy** [comment]
Yeah you should get it from here: https://github.com/iNDS-Team/iNDS/releases

**Comment by FrederickGeek8** [comment]
BuildStore is an acceptable place to get it from if you don't have access to app signing (through Apple Developer). Normally if you can't open the app, that is something that BuildStore needs to fix.
